relayer-cli/src/commands/query/connection.rs

```diff
@@ -22,9 +22,6 @@ pub struct QueryConnectionEndCmd {
 
     #[options(free, required, help = "identifier of the connection to query")]
     connection_id: ConnectionId,
-
-    #[options(help = "height of the state to query", short = "h")]
-    height: Option<u64>,
```

Comment:
@ancazamfir Is that ok with you, or do you sometimes rely on being able to query the connection end at specific heights?